.style.yapf

```diff
@@ -15,7 +15,7 @@ SPACES_BEFORE_COMMENT=2
 SPACE_BETWEEN_ENDING_COMMA_AND_CLOSING_BRACKET=False
 SPLIT_ARGUMENTS_WHEN_COMMA_TERMINATED=False
 SPLIT_BEFORE_BITWISE_OPERATOR=True
-SPLIT_BEFORE_FIRST_ARGUMENT=True
+SPLIT_BEFORE_FIRST_ARGUMENT=False
```

Comment:
Why? Turning this off creates harder to read constructions like:
+    artifact_definition = artifact.ArtifactDefinition(name,
 -        name, description=description)
+                                                      description=description)

where the treat this as a regular text (sentence) approach is much friendlier (less eye movement required).